Unix/tools/auth_tests_setup.sh

```diff
@@ -37,34 +40,60 @@ do
 done
 
 #
-# if required env variables don't exist, then quit
+# Setup the NTLM auth file for client and server. They must be different files because 
+# if the differeing permissions, as the server file must be owned by root and the client by
```

Comment:
Out of curiosity, must the file be owned by root? Do you validate that? What if we move to a non-prived OMI server? Is it acceptable to be owned by the OMI server account, in that case, or would we need a code change?